main/options/options.cc

```diff
@@ -627,6 +627,8 @@ buildOptions(const vector<pipeline::semantic_extension::SemanticExtensionProvide
                                  "Packages which are allowed to ignore the restrictions set by `visible_to` "
                                  "and `export` directives",
                                  cxxopts::value<vector<string>>(), "<name>");
+    options.add_options(section)("packager-layer", "Valid layer names for packages.",
```

Comment:
Not a huge fan of this flag name, but couldn't think of anything better, so open to suggestions.

Reply:
What do you not like about the current name?

Reply:
I think it's mainly that it doesn't really convey that these are the _possible_ layers, and not the layer being assigned to a package. Maybe `--packager-possible-layer` or `--packager-valid-layer` is better?

It's also a bit weird to me that it's singular given that it is called like `--packager-layer=a,b,c` (ie. with a list of items), but adding an `s` at the name might be weird when called in the repeated form: `--packager-layers a --packager-layers b`